src/main/app.js

```diff
@@ -147,10 +145,7 @@ app.on('ready', async () => {
   // When user closes all windows
   // On Windows, the command practice is to quit the app.
   app.on('window-all-closed', () => {
-    if (
-      appTriedToQuit ||
```

Comment:
Why is this deleted? this is important for saving unsaved data. Check the rest of the file for this `appTriedToQuit`

Reply:
I added appTriedToQuit again, and it still works, the checks also passed now. The problem seems to be `!getSetting(store.getState(), 'isTrayIconEnabled'`

Reply:
@rwolf449 Then we have to figure out a way to fix it, not remove it. Since all OS's rely on this setting to NOT quit if the tray icon is enabled.